components/ipfs/ipfs_utils_unittest.cc

```diff
@@ -53,6 +53,30 @@ class IpfsUtilsUnitTest : public testing::Test {
   GURL public_gateway_;
 };
 
+TEST_F(IpfsUtilsUnitTest, CIDValidation) {
```

Comment:
Add tests for:

- inlined CID like `bafkqaaa`
- base36 with libp2p-key codec like `k51qzi5uqu5dgutdk6i1ynyzgkqngpha5xpgia3a5qqp4jsh0u4csozksxel2r`